pkg/provider/helm/values.go

```diff
@@ -158,26 +169,57 @@ func (g *HelmValuesGenerator) GenerateValues() (map[string]any, error) {
 		"enabled": false,
 	}
 
+	cses, err := getOrCreateNestedMap(identities, "clusterStaticEntries")
+	if err != nil {
+		return nil, fmt.Errorf("failed to get clusterStaticEntries map from identities: %w", err)
+	}
+
 	filter := &datasourcepb.ListAPBindingsRequest_Filter{TrustZoneName: &g.trustZone.Name}
 	bindings, err := g.source.ListAPBindings(filter)
 	if err != nil {
 		return nil, fmt.Errorf("failed to list attestation policy bindings: %w", err)
 	}
 
-	// Adds the attestation policies as ClusterSPIFFEID CRs to be reconciled by the spire-controller-manager.
+	staticAP := false
+
+	// Adds the attestation policies as either ClusterSPIFFEID or ClusterStaticEntry CRs to be reconciled by the spire-controller-manager.
 	for _, binding := range bindings {
 		// nolint:staticcheck
 		policy, err := g.source.GetAttestationPolicy(binding.Policy)
 		if err != nil {
 			return nil, err
 		}
 
-		clusterSPIFFEIDs, err := attestationpolicy.NewAttestationPolicy(policy).GetHelmConfig(g.source, binding)
-		if err != nil {
-			return nil, err
+		if _, ok := policy.Policy.(*attestation_policy_proto.AttestationPolicy_Kubernetes); ok {
+			clusterSPIFFEID, err := attestationpolicy.NewAttestationPolicy(policy).GetHelmConfig(g.source, binding)
+			if err != nil {
+				return nil, err
+			}
+
+			csids[policy.Name] = clusterSPIFFEID
+		} else if _, ok := policy.Policy.(*attestation_policy_proto.AttestationPolicy_Static); ok {
+			clusterStaticEntry, err := attestationpolicy.NewAttestationPolicy(policy).GetHelmConfig(g.source, binding)
+			if err != nil {
+				return nil, err
+			}
+
+			staticAP = true
+
+			cses[policy.Name] = clusterStaticEntry
 		}
+	}
 
-		csids[policy.Name] = clusterSPIFFEIDs
+	// Adds a ClusterStaticEntry CR for the SPIRE agents, so that the parent ID is deterministic.
```

Comment:
I'm not familiar here with the implications of this, but are there any second order effects of doing this beyond deterministic for the static entry?

Reply:
Just 2 more from me:
* Any benefit to making the format of that SPIFFE ID given to the agent via the StaticEntry?
* Does this need to be flagged as a potential side effect of using `APStatic` in a warning or similar from `cofidectl` use?

Reply:
They are good questions! I think the alias SPIFFE ID created for the SPIRE agent(s) to facilitate static attestation policy is maybe something users could define but it not something that should be set via an attestation policy. The static attestation policy should just reference it. The current approach in effect manages that for the user with some opinion, as we do elsewhere I guess.

I think a warning could be good, to make users aware that an additional alias SPIFFE ID will be created if they choose to use a static attestation policy?

Reply:
Agreed on first point, maybe just addition of a notification/warning here is good for now

Reply:
I've added this warning message:
```
cofidectl attestation-policy add static --name foo-static --spiffeid spiffe://foo.bar/workloads --selectors k8s:ns:test

Warning: Creating a static attestation policy neccessitates the creation of an additional alias registration entry for SPIRE agent(s).
This means that each SPIRE agent will receive the same SPIFFE ID.
Do you want to continue? [y/N]:
```